torch/csrc/autograd/engine.h

```diff
@@ -26,6 +27,8 @@ struct ReadyQueue;
 
 namespace torch { namespace autograd {
 
+using FutureVariableList = torch::utils::Future<variable_list>;
```

Comment:
nit: I personally find the rhs clearer than the type alias.

Reply:
This would actually look like this in the signature: `std::shared_ptr<torch::utils::Future<variable_list>>`. I felt it was more compact writing just `std::shared_ptr<FutureVariableList>`.

Reply:
In speech, I'd also definitely refer to this as a "VariableListFuture" over "FutureVariableList", because "VariableList" tells me a lot more about the type than Future.